data_processors/pipeline/orchestration/umccrise_step.py

```diff
@@ -81,10 +79,8 @@ def prepare_umccrise_jobs(this_workflow: Workflow) -> List[Dict]:
 
     job = {
         "dragen_somatic_directory": dragen_somatic_directory,
-        "fastq_list_rows_germline": fqlr_germline,
-        "output_directory_germline": normal_rgsm,
+        "dragen_germline_directory": dragen_germline_directory,
```

Comment:
Looks like, input structure has changed a bit. This job struct will have to have a binding contract with the job trigger Lambda counterpart.

https://github.com/umccr/data-portal-apis/blob/344a7936503ab8f9ae276ce8da5c09eec5a18dc9/data_processors/pipeline/lambdas/umccrise.py#L62-L90

Perhaps, this has to stem from input template JSON changes; discussed upstream at infra repo. ditto

https://github.com/umccr/infrastructure/pull/284#pullrequestreview-1327229613

Reply:
Updated the event payload in the documentation for umccrise, should be able to resolve this conversation